routesrv/eskipbytes.go

```diff
@@ -51,7 +59,8 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer span.Finish()
 
 	if data, initialized := e.bytes(); initialized {
-		w.Write(data)
+		w.Header().Add("Etag", e.etag)
+		http.ServeContent(w, r, "", time.Time{}, bytes.NewReader(data))
```

Comment:
https://pkg.go.dev/net/http#ServeContent
> If the response's Content-Type header is not set, ServeContent first tries to deduce the type from name's file extension and, if that fails, falls back to reading the first block of the content and passing it to DetectContentType.

Lets set content type explicitly to disable detection overhead